### ISSUE TITLE
Error: No style defined for element

### ISSUE THREAD

**Issue by mathias-jakobsen** [iantrich/config-template-card#52]
**Checklist:**

- [x ] I updated to the latest version available
- [x ] I cleared the cache of my browser

**Release with the issue:** 
0.117.5

**Last working release (if known):**

**Browser and Operating System:** 
Microsoft Edge

**Description of problem:** 
When config template card is used as an element in a picture-elements card, I get an error saying "No style is defined for element".

Code used:
```
 - type: "custom:config-template-card"
   entities:
     - "light.bathroom_spotlights"
   element:
     type: image
     image: "${states['light.bathroom_spotlights'].attributes.hs_color ? '/local/floor_plan/bathroom.png?v=51' : '/local/floor_plan/bathroom.png?v=51'}"
     hold_action:
       action: none
     tap_action:
       action: none
   style:
     filter: "${`hue-rotate(${states['light.bathroom_spotlights'].attributes.hs_color ? states['light.bathroom_spotlights'].attributes.hs_color[0] : 0}deg)`} saturate(100%)"
     height: 100%
     left: 50%
     mix-blend-mode: lighten
     opacity: "${states['light.bathroom_spotlights'].attributes.brightness / 255}"
     top: 50%
     width: 100%`
```    

**Javascript errors shown in the web inspector (if applicable):** 
element custom:config-template-card Error: No style defined for element
    at HTMLElement.setConfig (config-template-card.js:3271)
    at s (chunk.cdf43a32e306acfdb2f4.js:1749)
    at chunk.cdf43a32e306acfdb2f4.js:1749
    at d (chunk.cdf43a32e306acfdb2f4.js:1749)
    at l (chunk.cdf43a32e306acfdb2f4.js:1749)
    at pe (chunk.070388aef4afe7ef2d22.js:1049)
    at n (chunk.070388aef4afe7ef2d22.js:1)
    at chunk.070388aef4afe7ef2d22.js:1
    at Array.forEach (<anonymous>)
    at HTMLElement.setConfig (chunk.070388aef4afe7ef2d22.js:1)
```

```

**Additional information:**

**Comment by iantrich** [comment]
For now put it at the element level as well. But I don't think all things will work currently as everything will be applied to the config-template-card and not the inner element. So some things like filter will likely not work. Might have to look at this again as I did put it together pretty quickly and didn't test all that much besides placement.

**Comment by vring0** [comment]
![image](https://user-images.githubusercontent.com/34909289/126989817-85c418c5-ff4b-4594-b289-389d7d62d5fb.png)
Issue has been around for a long time, but hasn't been closed. I took your example from Readme and it doesn't work on my plan. Where is the mistake?

**Comment by iantrich** [comment]
Try putting your style in the element as well; I'm guessing the config check is more strict now

**Comment by iantrich** [comment]
fixed in 1.3.4
